src/main/resources/META-INF/plugin.xml

```diff
@@ -21,11 +21,12 @@
       0.3.1: Release for IntelliJ 14.1 (unchanged functionality)
       0.3.2: Release for IntelliJ 15.0 (unchanged functionality)
       0.3.3: Release for IntelliJ 15.0.2, scala-plugin 2.0.4, support inheriting step definitions from traits (#16)
+      0.3.4: Release for IntelliJ 2016.1 (scala-plugin 3.0.0)
     ]]>
     </change-notes>
 
     <!-- please see http://confluence.jetbrains.com/display/IDEADEV/Build+Number+Ranges for description -->
-    <idea-version since-build="143.1184" until-build="145.0"/>
+    <idea-version since-build="143.1184" until-build="146.0"/>
```

Comment:
The corresponding wiki page is not the clearest ever but I followed the same schema you had before and put the current 2016.1 build (145.258)